QC suggestions for RC1 (some minor clarifications) (#71)
riscv/riscv-smmtt: chapter4.adoc

|`2M_PAGES` a|
This encoding applies only to RV64.
_The 32 MiB range of address space is partitioned into 16 2 MiB pages where each

> page has read/write/execute/access disallowed/not specified via the `INFO` field._

Reviewer: ```suggestion
page has read/write/execute access specified via the `INFO` field._
```

|`4M_PAGES` a|
This encoding applies only to RV32.
_The 32 MiB range of address space is partitioned into 8 4 MiB pages where each

> page has read/write/execute/access disallowed/not specified via the `INFO` field._

Reviewer: ```suggestion
page has read/write/execute access specified via the `INFO` field._
```

[NOTE]
====

> Ongoing MTT lookups are not required to complete or terminate when `mttp` is 
> effectively disabled, unless a `MFENCE.SPA` instruction

Reviewer: ```suggestion
MTT lookups that began while `mttp` was active are not required to complete or
terminate when `mttp` is no longer active, unless a `MFENCE.SPA` instruction
```

Reviewer: reverting this change - following the same language as the priv spec for vs and g stage lookups.
